static/js/components/subject_page/block_container.tsx

```diff
@@ -99,7 +100,12 @@ export function BlockContainer(props: BlockContainerPropType): JSX.Element {
     const result: Item[] = [];
     for (const prop in svExtendedMap) {
       const urlSv = svExtendedMap[prop].join(DELIM);
-      const url = `/explore/#t=${urlSv}&p=${exploreData.place}&pcmp=${exploreData.cmpPlace}&pt=${exploreData.placeType}&dc=${exploreData.dc}&em=1`;
+      const url = `/explore/#${getUpdatedHash({
+        t: urlSv,
+        p: exploreData.place,
+        q: "",
+        em: "1",
+      })}`;
```

Comment:
is DC no longer passed here?

Reply:
getUpdatedHash will keep all params the same and update the ones you pass in